### DISCUSSION TITLE
Wat verstaan we onder het bijhouden van lifecycle management?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#36] [Q&A] [answered]
Van #2 is er een vraag voor het bijhouden van lifecycle management, als we kijken naar de Common Ground componenten catalogus is dat momenteel relatief simpel ingericht. De levenscycli wordt bijgehouden door middel van een keuze m.b.t. de status.

![Componenten Catalogus Component bewerken](https://user-images.githubusercontent.com/4021899/170700612-3c9bf125-2ba6-4f64-abba-5f4cb0b96243.png)

Vinden we dat voor OpenCatalogi fase 1 voldoende of willen we meer dingen in de trant van semantic versioning.

**Comment by Makkmetp** [comment]
Betreft het de levencycli van het component ansich of gaat dit over de levenscycli van dat een component dat door een organisatie in gebruik is?

**Comment by rubenvdlinde** [comment, marked as answer]
Zie ook #81, dit gaat een stuk verder dan de status die componentencatalogus nu kent.

**Reply by RonaldvCortenberghe** [reply]
is akkoord
